rules/no-inline-units/no-inline-units.test.js

```diff
@@ -0,0 +1,83 @@
+const { RuleTester } = require("eslint");
```

Comment:
I do not find any TS-tests. Should we add it?

Reply:
I can add TS-tests, but I don't understand what specific cases they should cover.
Can u help me with that?

Reply:
In mind, we should cover something like `sample({ source: createCustomStoreBySomeFactory() })` where `createCustomStoreBySomeFactory` is not imported from `effector`, but it returns `Store`.